Python/PR_Analyzer/main.py

```diff
@@ -0,0 +1,381 @@
+#Necessary Packages
+import datetime
+import json
+import re
+
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+from jupyterthemes import jtplot
+from matplotlib import pyplot as plt
+from sklearn.cluster import KMeans
+
+jtplot.style('monokai','notebook',ticks=True)
```

Comment:
Do we need this? I don't think so.

Reply:
I just used it for making the plots look a little themed. I will remove it.